.github/workflows/libosrm.yml

```diff
@@ -0,0 +1,61 @@
+name: Libosrm CI
+on:
```

Comment:
Would be nice to add a `tags` entry in order to trigger a build for all RC and final versions. The current Travis pattern is `/^v\d+\.\d+(\.\d+)?(-\S*)?$/` which is probably overly complicated, anything like `v*.*.*` would probably do.

Reply:
Added tags trigger.